helm-chart/configs/grafana-top-dashboard.json

```diff
@@ -310,7 +310,7 @@
           "alertThreshold": true
         },
         "percentage": false,
-        "pluginVersion": "8.1.7",
```

Comment:
[this grafana dashboard ](https://github.com/nginxinc/nginx-service-mesh/blob/main/examples/grafana/nginx-service-mesh-summary.json) needs to be updated as well

Reply:
Okay.  Thanks!

Reply:
I just updated all of the pluginVersion fields in those files.

Reply:
I'm sorry, missed one: `examples/grafana/nginx-mesh-top.json`

Reply:
Oh.  Thanks!

Reply:
Can you tell me which line I missed?  I don't see it.

Reply:
Oh, strange. Guess something failed to update on my end. It showed the old version before, but now it's showing the correct one. Sorry about that!

Reply:
Oh good.  Thanks!